packages/components/src/sandbox/index.native.js

```diff
@@ -322,6 +322,7 @@ function Sandbox( {
 				sandboxStyles[ 'sandbox-webview__container' ],
 				containerStyle,
 			] }
+			injectedJavaScript={ observeAndResizeJS }
```

Comment:
As noted above, we could replace this with:

` injectedJavaScript={ customJS || observeAndResizeJS } `

But I'm not sure why we would want to override the resize behavior.

Reply:
As I mentioned above, I'd lean toward going this way for the `customJS` too.

Reply:
Updated in 2fb6eb3eff1fc4f217a26901a4248a84401956b1